src/components/Input/TextArea/TextArea.tsx

```diff
@@ -0,0 +1,31 @@
+import { cx } from '@emotion/css';
+import React, { PropsWithChildren } from 'react';
+import { TextAreaProps } from './TextAreaProps';
+import { textAreaStyle, stySizeDefault, styStatusDefault } from './styles';
+
+const TextArea = (props: PropsWithChildren<TextAreaProps>): JSX.Element => {
+    const { placeholder, text, txtAreaSize, txtAreaStatus } = props;
+
+    const inputSize =
+        txtAreaSize !== undefined
+            ? `${txtAreaSize} ${textAreaStyle}`
+            : cx(stySizeDefault);
+
+    const inputStatus =
+        txtAreaStatus !== undefined
+            ? `${txtAreaStatus} ${textAreaStyle}`
+            : cx(styStatusDefault);
```

Comment:
sebenernya ada cara yang lebih bagus, but for now no worries.. karena jelasinnya juga susah wkwk